TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/DrivebaseAuto.java

```diff
@@ -0,0 +1,91 @@
+
+package org.firstinspires.ftc.teamcode.opmodes;
+
+import com.acmerobotics.roadrunner.TrajectoryActionBuilder;
+import com.arcrobotics.ftclib.command.CommandScheduler;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.acmerobotics.roadrunner.Action;
+import com.acmerobotics.roadrunner.Pose2d;
+import com.acmerobotics.roadrunner.SequentialAction;
+import com.acmerobotics.roadrunner.Vector2d;
+import com.acmerobotics.roadrunner.ftc.Actions;
+import org.firstinspires.ftc.teamcode.roadrunner.MecanumDrive;
+
+import org.firstinspires.ftc.teamcode.*;
+
+@Autonomous(name="Drive-Base-Auto", group = "Real")
+public class DrivebaseAuto extends LinearOpMode {
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+
+        Robot robot = new Robot(this);
+
+        CommandScheduler.getInstance().reset();
+        CommandScheduler.getInstance().cancelAll();
+
+
+        int allianceNumber = 1;//blue = 1, red = -1
+
+        //tell the bot where it starts
+        Pose2d initialPose = new Pose2d(-3 * Constants.INCH_TO_TILE * allianceNumber, 1.5 * Constants.INCH_TO_TILE * allianceNumber, Math.toRadians(90 * allianceNumber));
+        MecanumDrive drive = new MecanumDrive(hardwareMap, initialPose);
+
+        //decide where we should park
+        boolean isGettingNeutral = true;
+        boolean isTouchingLowRung = false;
```

Comment:
Again, you don't need all of this flexibility because it's not like the drivers can just change hard-coded values from the gamepad. You'll need separate opModes.

Reply:
Sure, this is just for test and I will just copy and edit more once this is tested